daisy/instance.go

```diff
@@ -182,7 +182,7 @@ func (i *Instance) populateMetadata(w *Workflow) dErr {
 	i.Metadata["daisy-outs-path"] = "gs://" + path.Join(w.bucket, w.outsPath)
 	if i.StartupScript != "" {
 		if !w.sourceExists(i.StartupScript) {
-			return errf("bad value for StartupScript, source not found: %s", i.StartupScript)
+			return Errf("bad value for StartupScript, source not found: %s", i.StartupScript)
```

Comment:
StartupScript is not private so no need to hide it here

Reply:
It's a URL, which exposes some info, e.g. bucket name, which is PII